src/core/index.ts

```diff
@@ -63,27 +71,106 @@ const approveChange = async (context: Context): Promise<void> => {
  * @param context The request context created by Probot core.
  * @param startTime The timestamp that this run started.
  */
-const createPassingStatus = async (
+export const createPassingStatus = async (
   context: Context,
   startTime: string,
 ): Promise<void> => {
+  await createStatus(
+    context,
+    "success",
+    "completed",
+    composeStatusCheckTitle(),
+    composeStatusCheckSummary(),
+    composeStatusCheckDetails(),
+    startTime,
+  );
+};
+
+/**
+ * Create a status in pull requets indicating that the run
+ * has started.
+ *
+ * @param context The request context from Probot core.
+ * @param startTime The string time that the run started.
+ */
+export const createStartStatus = async (
+  context: Context,
+  startTime: string,
+): Promise<void> => {
+  await createStatus(
+    context,
+    undefined,
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092518)